Packs/EDL/Integrations/EDL/EDL.py

```diff
@@ -330,6 +335,9 @@ def create_values_for_returned_dict(iocs: list, request_args: RequestArguments)
                 # invalid tokens in indicator- if drop_invalids is set - ignore the indicator
                 if request_args.drop_invalids:
                     continue
+            # if the indicator is a URL without a slash, we add a slash at the end
+            if request_args.url_add_slash and '/' not in indicator:
```

Comment:
@jtschichold this will affect all indicator types that are not `IP/CIDR` is this the intended result?
If this is meant for a specific type of indicator (such as URL) you can use the `ioc_type`. Otherwise, please ignore my comment.

Reply:
The url_add_slash option should be applied with the same logic as url_port_stripping option.
The existing code is applying url_port_stripping option to all the IOC that are not IPs/CIDRs.
In theory both url_add_slash should only be applied to URL, Domain and DomainGlob.
Please also check my comment on the PR about possible issue with existing EDL. Maybe the add_slash option should be set to false by default?

Reply:
@jtschichold I feel like the previous behaviour wasn't intended, seeing as it refers to URLs in the docs (@hod-alpert might know better). I won't ask to change the existing behaviour, but I think that going forward we should enforce a stricter check for `indicator type transformations`.
Please set it to `default: false` as you mentioned, since this might break BC.

@anara123 fyi